queue_services/entity-filer/src/entity_filer/filing_processors/change_of_registration.py

```diff
@@ -31,29 +31,32 @@
 
 
 def process(
-    legal_entity: LegalEntity,
+    business: any,
     change_filing_rec: Filing,
     change_filing: Dict,
     filing_meta: FilingMeta,
 ):
```

Comment:
just a general thing about the change of registration processor.  I think we need to follow how conversion and registration filing currently processes offices.  i.e. delete all existing offices first and recreate them.

Reply:
Some updates. And broken unit tests will be fixed in the comming commits.

Reply:
Should be fixed now.